src/main/resources/assets/createcasing/lang/ru_ru.json

```
@@ -1,232 +1,489 @@
+  "block.createcasing.railway_encased_crimson_cogwheel": "Багровая шестерня в железнодорожном корпусе",
+  "block.createcasing.railway_encased_crimson_large_cogwheel": "Большая багровая шестерня в железнодорожном корпусе",
   "block.createcasing.railway_encased_crimson_shaft": "Багровый вал в железнодорожном корпусе",
+  "block.createcasing.railway_encased_dark_oak_cogwheel": "Шестерня из тёмного дуба в железнодорожном корпусе",
+  "block.createcasing.railway_encased_dark_oak_large_cogwheel": "Большая шестерня из тёмного дуба в железнодорожном корпусе",
   "block.createcasing.railway_encased_dark_oak_shaft": "Вал из тёмного дуба в железнодорожном корпусе",
   "block.createcasing.railway_encased_glass_shaft": "Стеклянный вал в железнодорожном корпусе",
+  "block.createcasing.railway_encased_jungle_cogwheel": "Шестерня из тропического дерева в железнодорожном корпусе",
+  "block.createcasing.railway_encased_jungle_large_cogwheel": "Большая шестерня из тропического дерева в железнодорожном корпусе",
   "block.createcasing.railway_encased_jungle_shaft": "Вал из тропического дерева в железнодорожном корпусе",
   "block.createcasing.railway_encased_large_cogwheel": "Большая шестерня в железнодорожном корпусе",
+  "block.createcasing.railway_encased_mangrove_cogwheel": "Мангровая шестерня в железнодорожном корпусе",
+  "block.createcasing.railway_encased_mangrove_large_cogwheel": "Большая мангровая шестерня в железнодорожном корпусе",
   "block.createcasing.railway_encased_mangrove_shaft": "Мангровый вал в железнодорожном корпусе",
   "block.createcasing.railway_encased_mldeg_shaft": "Вал Mldeg в железнодорожном корпусе",
+  "block.createcasing.railway_encased_oak_cogwheel": "Дубовая шестерня в железнодорожном корпусе",
+  "block.createcasing.railway_encased_oak_large_cogwheel": "Большая дубовая шестерня в железнодорожном корпусе",
   "block.createcasing.railway_encased_oak_shaft": "Дубовый вал в железнодорожном корпусе",
   "block.createcasing.railway_encased_shaft": "Вал в железнодорожном корпусе",
   "block.createcasing.railway_encased_spruce_shaft": "Еловый вал в железнодорожном корпусе",
+  "block.createcasing.railway_encased_warped_cogwheel": "Искажённая шестерня в железнодорожном корпусе",
+  "block.createcasing.railway_encased_warped_large_cogwheel": "Большая искажённая шестерня в железнодорожном корпусе",
   "block.createcasing.railway_encased_warped_shaft": "Искажённый вал в железнодорожном корпусе",
   "block.createcasing.railway_gearbox": "Железнодорожная коробка передач",
   "block.createcasing.railway_mixer": "Железнодорожный смешиватель",
   "block.createcasing.railway_press": "Железнодорожный пресс",
+  "block.createcasing.refined_radiance_adjustable_chain_gearshift": "Сиящая регулируемая цепная коробка передач",
+  "block.createcasing.refined_radiance_configurable_gearbox": "Сиящая настраиваемая коробка передач",
+  "block.createcasing.refined_radiance_depot": "Сиящее депо",
```

The letter "ю" is lost here and in the two lines above:
```suggestion
  "block.createcasing.refined_radiance_depot": "Сияющее депо",
```

---

O-o-ops :P